lib/OpenQA/Worker.pm

```diff
@@ -104,6 +107,27 @@ sub catch_exit {
     }
 }
 
+sub setup_logging {
+    my ($app) = @_;
+
+    my $logfile = $ENV{OPENQA_WORKER_LOGFILE} || $worker_settings->{LOG};
+
+    if ($logfile) {
+        $app->log->path("$logfile.$instance");    # So each worker get it's own log
```

Comment:
Not sure, maybe you meant `path($logfile.$instance)` here?

Reply:
@mudler 

I want with the dot in the middle. Not just concatenation

Reply:
just a personal opinion here, i really dislike logs not ending with ".log" but it's not my call :)

Reply:
I'm ok with

- {logfile}-{instance}.log

as i hate chars that i need to escape on the console :-P